AmplifyPlugins/API/Podfile.lock

```diff
@@ -33,13 +35,12 @@ PODS:
   - AWSPluginsCore (1.5.1):
     - Amplify (= 1.5.1)
     - AWSCore (~> 2.20.0)
-    - AWSMobileClient (~> 2.20.0)
   - CwlCatchException (1.0.2)
   - CwlPreconditionTesting (1.1.1):
     - CwlCatchException
   - Starscream (3.1.1)
   - SwiftFormat/CLI (0.44.17)
-  - SwiftLint (0.41.0)
+  - SwiftLint (0.42.0)
```

Comment:
can we apply `SwiftLint` related changes in a different PR?

Reply:
This comes automatically when I do pods update.